### PR TITLE
Update lavalink-without-ssl.md

### PR DESCRIPTION
**Pull request by growtoups**
Added my lavalink.

### CODE REVIEW

File: docs/NoSSL/lavalink-without-ssl.md [hunk 1]
```diff
@@ -13,7 +13,14 @@ description: Without SSL should work out of the box!
 
     v4 introduce a breaking changes that affects all library that are using v3 API.
     You must update your bot library to support v4!
-
+### Hosted by @ [Growtoups](https://micium-hosting.com)
+Version 3.7.10
+```bash
+Host : lavalink.micium=hosting.com
```
Comment (DarrenOfficial) [inline docs/NoSSL/lavalink-without-ssl.md:19]: this server does not work

**Review by DarrenOfficial — CHANGES_REQUESTED** [review]

File: docs/NoSSL/lavalink-without-ssl.md (hunk 1 above)
Comment (growtoups) [reply docs/NoSSL/lavalink-without-ssl.md:19]: it does work tho

Comment (growtoups) [reply docs/NoSSL/lavalink-without-ssl.md:19]: OH wait i have put a = instead of - fk

Comment (growtoups) [reply docs/NoSSL/lavalink-without-ssl.md]: aight it should be fixed

**Comment by growtoups** [comment]
i'll just make a new pull request

**Pull request closed without merging**